contributing.md

```diff
@@ -55,3 +92,14 @@ You may want to read this for a more detailed explanation (and links to other po
 how to write a good commit message).
 This content is licensed under [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).
 
+## How to reach us
+The mailing list for developers : systemroles@lists.fedorahosted.org
+
+[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+
+[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
+
+If you are using IRC, join the `#systemroles` IRC channel on freenode
```

Comment:
Please add a link to freenode, maybe https://freenode.net/kb/answer/chat

Reply:
I added this, thanks